lib/matplotlib/backends/backend_ps.py

```diff
@@ -651,8 +651,9 @@ def draw_text(self, gc, x, y, s, prop, angle, ismath=False, mtext=None):
                 kern = font.get_kern_dist_from_name(last_name, name)
                 last_name = name
                 thisx += kern * scale
-                xs_names.append((ps_name, thisx, name))
```

Comment:
I guess(?) that should just have been stream.append((ps_name, thisx, name)) and xs_names is unused and can be removed?

Reply:
Yes that's what I wondered and removed `ps_name` from the `xs_names` append and appended them into the stream in a fresh line

Reply:
I could remove xs_names completely if its not of any significance here